qa/workunits/fs/quota/quota.sh

```diff
@@ -106,8 +106,8 @@ expect_false setfattr -n ceph.quota.max_bytes -v -9223372036854775809 .
 
 setfattr -n ceph.quota.max_bytes -v 0 .
 setfattr -n ceph.quota.max_bytes -v 1Ti .
-setfattr -n ceph.quota.max_bytes -v 8388608Ti .
-expect_false setfattr -n ceph.quota.max_bytes -v 8388609Ti .
+setfattr -n ceph.quota.max_bytes -v 8388607Ti .
+expect_false setfattr -n ceph.quota.max_bytes -v 8388608Ti .
```

Comment:
thanks for fixing this

do you think we should warn users while taking overflowing input?

Reply:
warn where? logs? that's already done.

If you are talking about via the CLI itself, maybe have a mention in the docs.

Reply:
add a note in docs warning users not to exceed? looks quite trivial there

Reply:
I'm not clear the warning you mention. We do return EINVAL and dump in the logs.

Reply:
return a warning "Integer overflow, please resort to acceptable range", something like this on CLI

Reply:
Maybe. Let's do it as a separate change.

Reply:
yeah sure, i'll close this conversation. will approve the pr